### PR TITLE
[ Sample2 ] First Change - Pull Request Feature

### PR DESCRIPTION
**Pull request by zaki2030**
some changes applied to initiate a pull request.

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ this is a sample file
 
 
 A new dummy change
+
```
Comment (zaki2027) [inline test.txt:5]: hello zaki2030, good you performed this change

File: test1.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 this is a new file
 
 adding 2nd change commit...
+
+again changing another file.
```
Comment (zaki2027) [inline test1.txt:5]: :)

**Review by zaki2027 — CHANGES_REQUESTED** [review]
please, apply those changes

File: test1.txt (hunk 2 above)
Comment (zaki2030) [reply test1.txt:5]: :))

**Review by zaki2027 — APPROVED** [review]
changes have been reviewed and accepted

**Pull request merged**